src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/dataPrivacyRules.tsx

```diff
@@ -90,55 +92,86 @@ class DataPrivacyRules extends React.Component<Props, State> {
 
   api = new Client();
 
+  loadOrganizationRules = () => {
+    const {organization} = this.props;
+
+    if (this.isProjectLevel()) {
+      try {
+        const convertedRules = this.convertRelayPiiConfig(organization.relayPiiConfig);
+        this.setState({
+          orgRules: convertedRules,
+        });
+      } catch {
+        addErrorMessage(t('Unable to load organization rules'));
+      }
+    }
+  };
+
   loadRules() {
     try {
-      const relayPiiConfig = this.state.relayPiiConfig;
-      const piiConfig = relayPiiConfig ? JSON.parse(relayPiiConfig) : {};
-      const rules: PiiConfigRule = piiConfig.rules || {};
-      const applications: Applications = piiConfig.applications || {};
-      const convertedRules: Array<Rule> = [];
-
-      for (const application in applications) {
-        for (const rule of applications[application]) {
-          if (!rules[rule]) {
-            if (rule[0] === '@') {
-              const [type, method] = rule.slice(1).split(':');
-              convertedRules.push({
-                id: convertedRules.length,
-                type: type as RuleType,
-                method: method as MethodType,
-                source: application,
-              });
-            }
-            continue;
-          }
-
-          const resolvedRule = rules[rule];
-          if (resolvedRule.type === RuleType.PATTERN && resolvedRule.pattern) {
-            const method = resolvedRule?.redaction?.method;
+      const convertedRules = this.convertRelayPiiConfig(this.state.relayPiiConfig);
+      this.setState({
+        rules: convertedRules,
+        savedRules: convertedRules,
+      });
+    } catch {
+      addErrorMessage(t('Unable to load project rules'));
+    }
+  }
 
+  isProjectLevel = () => this.props.endpoint.includes('projects');
```

Comment:
NB: this doesn't need to be a function, if you wanted to have it behave as before where you initialize it in state.

Reply:
yes I prefer a state, even if we don't update it afterwards .... I will update the code 😉

Reply:
I meant it could be this so you would not need the function call and it is computed just once

```
isProjectLevel = this.props.endpoint.includes('projects');
```